tendrl/gluster_integration/manager/manager.py

```diff
@@ -200,6 +266,27 @@ def on_pull(self, raw_data, cluster_id):
                     index += 1
                 except KeyError:
                     break
+            # poplate the volume options
+            reg_ex = re.compile("^volume[0-9]+.options+")
```

Comment:
So even if we move inside main loop of volume attributes, there also we would have to do at least the way its done for bricks and we wont be able to save no of writes to etcd.